manifests/controller-manager/cloud-controller-manager-roles.yaml

```diff
@@ -10,8 +10,10 @@ items:
     resources:
     - leases
     verbs:
-    - get
     - create
+    - get
+    - list
```

Comment:
THe commit message said it's move from somewhere ? so where its original place and why not remove them?
or if we add those list and watch, what's the reason behind it?

Reply:
@jichenjc Is your concern resolved?